test/viewmodels/ViewReturnSummaryViewModelSpec.scala

```diff
@@ -16,25 +16,38 @@
 
 package viewmodels
 
-import models.returns.{ChargeDetails, IdDetails, SubmittedReturn}
+import models.returns.{IdDetails, ReturnDisplayApi, ReturnDisplayChargeDetails, ReturnDisplayDetails}
 import org.scalatestplus.play.PlaySpec
 import viewmodels.checkAnswers.{Field, ViewReturnSummaryViewModel}
 
 class ViewReturnSummaryViewModelSpec extends PlaySpec {
 
+
   "apply" must {
     "build the Summary section" in {
-      val submittedReturn = SubmittedReturn("31 July 2022", IdDetails("pptRef", ""), Some(ChargeDetails("", "", 400, "5 July 2022")), None, None)
+      val returnDisplayChargeDetails = ReturnDisplayChargeDetails(
+        "21C2", Some("charge-ref"), "2022-04-01", "2022-06-30", "2022-07-03", "New"
+      )
+      val returnDetails = ReturnDisplayDetails(
+        1, 2, 3, 4, 5, 6, 7, 8, 9, 10
```

Comment:
Minor observation; Could this test data be more realistic, or maybe mixed up to include some decimal values too?

Reply:
Defo, yes